packages/language-server/src/completion/completions.ts

```diff
@@ -671,7 +775,7 @@ function getFieldType(line: string): string | undefined {
     return undefined
   }
   let type = wordsInLine[1]
-  if (type !== '' && type !== undefined) {
+  if (type.length !== 0 && type !== undefined) {
```

Comment:
Can `type` be `undefined`?

Reply:
Good point, no, it can't be undefined, since there is a check in the line above for that. I'll remove this check here.